upgrade50Plus.sh

```diff
@@ -171,16 +224,37 @@ install_grub() {
 }
 
 reboot_device() {
-    pretty "Rebooting ePC..." "Please wait while" "the ePC is rebooting." "Rebooting ePC..." "Please wait."
+    pretty "Rebooting ePC..." "Please wait a while" "the ePC is rebooting." "Rebooting ePC..." "Please wait."
+    executeAsRoot "reboot"
+    sleep 5
+
+    while true; do
+        rm /root/.ssh/known_hosts &> /dev/null;
+        if executeAsRoot "exit"; then
+            break
+        fi
+        sleep 1
+    done
 
+    pretty "" "Cleaning up..." "" "Cleaning Up..." ""
```

Comment:
merging sda4 and sda5 into a new home partition has nothing to do with "reboot_device()" so I propose to extract this into a new function and call it from main() as a new step after reboot_device() finished.